src/main/java/io/github/darealturtywurty/turtychemistry/mixin/AnvilMixin.java

```diff
@@ -0,0 +1,79 @@
+package io.github.darealturtywurty.turtychemistry.mixin;
+
+import io.github.darealturtywurty.turtychemistry.common.TurtyTags;
+import io.github.darealturtywurty.turtychemistry.common.block.entity.AnvilBlockEntity;
+import io.github.darealturtywurty.turtychemistry.core.init.BlockEntityInit;
+import net.minecraft.core.BlockPos;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.item.ItemEntity;
+import net.minecraft.world.entity.player.Inventory;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.Items;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.AnvilBlock;
+import net.minecraft.world.level.block.EntityBlock;
+import net.minecraft.world.level.block.FallingBlock;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.material.FluidState;
+import net.minecraft.world.phys.BlockHitResult;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(AnvilBlock.class)
+public final class AnvilMixin extends FallingBlock implements EntityBlock {
+    public AnvilMixin(BlockBehaviour.Properties pProperties) {
+        super(pProperties);
+    }
+
+    @Override
+    public boolean onDestroyedByPlayer(BlockState state, Level level, BlockPos pos, Player player, boolean willHarvest, FluidState fluid) {
+
+        if (level.getBlockEntity(pos) instanceof AnvilBlockEntity anvilMechanicBlock) {
+            level.addFreshEntity(
+                    new ItemEntity(level, pos.getX(), pos.getY(), pos.getZ(), anvilMechanicBlock.getItem()));
+        }
+        return super.onDestroyedByPlayer(state, level, pos, player, willHarvest, fluid);
+    }
+
+    @Inject(method = "use", at = @At("HEAD"), remap = false, cancellable = true)
```

Comment:
Why do you have `, remap = false`?

Reply:
because when I test it without it, it refuses to launch saying "no mapping for this method"

Reply:
🤔 That's not meant to happen.

Reply:
Can you send the full error you get in discord.